site-packages/integralstor/remote_monitoring.py

```diff
@@ -0,0 +1,179 @@
+import os
+import shutil
+import urllib
+import json
+from integralstor_utils import config, networking
+
+
+def _get_config_file_path():
+    server_config_file_path = None
+    try:
+        config_files_dir, err = config.get_config_files_dir()
+        if err:
+            raise Exception(err)
+        server_config_file_path = '%s/remote_monitoring_servers' % config_files_dir
+        if not os.path.isfile(server_config_file_path):
+            if not os.path.exists(config_files_dir):
+                os.makedirs(config_files_dir)
+            with open(server_config_file_path, 'w') as f:
```

Comment:
Since the function is called `_get_config_file_path()` and it's supposed to 'get' something, it must do only that. In this case, it's creating the file 'remote_monitoring_servers' if it doesn't already exist. It will be better if this function returns just the path and let the calling function decide how to handle it.